File: listas basica dandre/java/Persona.java
public class Persona {
    private String nombre;
    private int edad;
    private String sexo;

    public Persona(String nombre, int edad, String sexo) {
        this.nombre = nombre;
        this.edad = edad;
        this.sexo =sexo.toUpperCase();

    }

    public String getNombre() {
        return nombre;
    }

    public int getEdad() {
        return edad;
    }

    public String getSexo() {
        return sexo;
    }
    
    public String toString() {
        return nombre + ", " + edad + " anos, " +(sexo.equals("M")? "Mujer" : "Hombre");
    }
}

File: listas basica dandre/python/listas basica dandre/persona.py
class Persona:
    def __init__(self, nombre, edad, sexo):
        self.nombre = nombre
        self.edad = edad
        self.sexo = sexo  # "M" para mujer, "H" para hombre

    def __str__(self):
        tipo = "Mujer" if self.sexo == "M" else "Hombre"
        return f"{self.nombre}, {self.edad} años, {tipo}"
